src/mobile/apps/artwork/components/related_artworks/index.coffee

```diff
@@ -76,12 +76,13 @@ module.exports = ->
       artist_collection = new Artworks _.shuffle artwork.artist?.artworks
       related_collection = new Artworks _.shuffle artwork.layer?.artworks
 
-      relatedWorks = [
-        { collection: auction_collection, title: "Other Works from the Auction", typeName: context.__typename }
-        { collection: artist_collection, title: "Other Works by #{artwork.artist?.name}", typeName: 'artist', isArtist: true }
-        { collection: related_collection, title: "Related Works", typeName: 'related' }
-      ]
-
+      if context.is_open
```

Comment:
This confused me a little bit, but in the metaphysics-query above, we only include the fragments if the following logic holds. Instead of passing the sections here and ignoring them in the next section, we can just only include the ones we care about in this case.